plugins/guisupport/guisupport.cpp

```diff
@@ -67,6 +67,17 @@ Q_DECLARE_METATYPE(QSurface::SurfaceType)
 Q_DECLARE_METATYPE(QSurfaceFormat::FormatOptions)
 #endif
 
+#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
+static bool isAcceptableWindow(QWindow *w)
+{
+    return w
+            && w->isTopLevel()
+            && w->surfaceClass() != QSurface::Offscreen
+            && w->title() != QStringLiteral("Offscreen")
```

Comment:
Is Offscreen something special from that application or is it from Qt?
Maybe add a comment

Reply:
Fixed